docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,177 @@
 
-It's fine to create other subdirectories for new collections of assets as needed but you'll need to modify the push script to include them.
-
-The release script for static assets is `.github/workflows/upload-assets-to-cdn.yml`. 
+1. Build `fxa-auth`, `fxa-content`, `fxa-oauth`, `fxa-profile`, `fxa-verifier` in stage on Jenkins using git commit from cloudops-deployment PR and docker images referenced in deploy bug
 
-Icons and other assets are normally deployed with every release. If icons need to be deployed outside the regular release process, ask operations for help.
+1. If this is a push to production:
+    1. Ensure we have the appropriate QA signoffs. If major issues are found, a new patch is made and engineering will need to create a patch release
+    1. Deploy fxa-* to production
+    1. Close the initial deployment bug. It may be commented on later if there are patch releases in production.
 
-## FAQ
+1. Otherwise, if this is a staging tag, request QA on stage by posting in the bug.
 
-### What if the merge messes up the changelog?
+### Patch releases
 
-After merging but before pushing, you should check the changelog to make sure that the expected versions are listed and they're in the right order. If any are missing or the order is wrong, manually edit the changelog so that it makes sense, using the commit summaries from `git log --graph --oneline` to fill in any blanks as necessary.
+Patch (dot) releases are used between official releases to release code into staging or production that can't wait for the normal release cycle. What warrants a patch release may be evaluated on a case-by-case basis, but [S1 bugs][bug-severity] would likely drive one. If QA's signoff after testing a release on staging comes back yellow or red, we should check with engineering management for their recommended course of action, as we may need a dot release before pushing a release to production.
 
-Then `git add` those changes and squash them into the preceding merge commit using `git commit --amend`. Now you can push and the merged changelog will make sense.
+Whenever a dot release is planned, ping the team to let them know you'll be tagging `train xxx.y`, with a list of issues to be included in the release. Other engineers may have fixes that wouldn't warrant a dot release on their own, but would be nice to ride along if a patch release is being tagged anyway.
 
-### What happens if there are merge conflicts (train-xxx => main)?
-Conflicts are most likely from a recently landed issue in `main`. Typically we create a new branch, resolve conflicts there, and then merge that branch into `main`.
+:::caution
+While it's oftentimes up to our own discretion, engineers should be **selective of fixes and mindful of asking** release owners to include their patches if the patch wouldn't justify a point release on its own, as it introduces more overhead.
```

Comment:
And risk of failures/issues.

Reply:
My personal opinions on what should and shouldn't warrant a dot release:

Examples that would warrant a dot release:
* A bug that prevents us from meeting legal obligations to FxA users/subscribers. E.g. all or some subscribers are no longer receiving invoice emails.
* S1s and S2s that significantly impact all or a plurality of users in production with no available workarounds. E.g. A RP just launched with tiered products, and PayPal users can't upgrade their subscriptions.
* Blocker QA fixes in the last sprint before a scheduled production launch.

Examples that would not warrant a dot release:
* Developer experience changes that go into effect on merge into `main`.
* Bugfixes that affect a very small minority of users in a narrow use case (<<1%) where workarounds are available.
* Bugfixes for features that are not currently being actively QAed in stage or enabled in production.
* Patches that simply didn't make the stage deploy cut-off for the sprint.

It might be worth opening this discussion up to the FxA/SubPlat team, though.

Reply:
For the most part, I agree with your examples.  I think it can be summed up with "S1s are the only bugs that come with the expectation of a dot release."  From your example, I would argue that a bug that significantly impacts all users in production with no workaround is an S1.

We may have rare special cases of needing to push off schedule to align with product launches or marketing campaigns, but they should be rare and well-discussed.

Reply:
I'll add 1-2 sentences along those lines^, as well as anything causing unexpected test failures once it reaches stage/prod, but maybe that last bit is obvious. It feels like we've had dot releases that weren't for bug fixes, but maybe I'm mostly thinking of a non-bug fix riding along.

Reply:
This all sounds good to me and definitely clears up some of my questions surrounding when a dot release is appropriate.

Reply:
@annasob @clouserw @biancadanforth 

Based on Anna's [comment here](https://github.com/mozilla/ecosystem-platform/pull/199/files), I used this phrasing:


> Patch (dot) releases are used between official releases to release code into staging or production that can't wait for the normal release cycle. What warrants a patch release may be evaluated on a case-by-case basis, but we may need one if:
> 
> - An [S1 bug][bug-severity] is found
> - We have legitimate (non-flaky, non-SRE related) test failures in staging or production
> - We need to land tasks to make a time-sensitive deliverable that didn't quite make the release, like a new RP launch
> 
> If QA's signoff after testing a release on staging comes back yellow or red, we should check with engineering management for their recommended course of action, as we may need a dot release before pushing a release to production.

I'm happy to change anything here. I almost added "and possibly S2s" but I know @clouserw you explicitely said "S1s are the only bugs that come with the expectation of a dot release" and it's sort of covered by the yellow/red QA signoff.